commands/peers/commands.go

```diff
@@ -0,0 +1,40 @@
+// Package peercommands implements the peer management commands
+package peercommands
+
+import (
+	"github.com/gluster/glusterd2/rest"
+)
+
+// Command is a holding struct used to implement the GlusterD Command interface
+type Command struct {
+}
+
+// Routes returns command routes. Required for the Command interface.
+func (c *Command) Routes() rest.Routes {
+	return rest.Routes{
+		rest.Route{
+			Name:        "GetPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}",
```

Comment:
Currently we are using the ID as a key in the store to store/retrieve the peer details. We'd definitely need to support name as key as that's a 3.x semantics but is there any specific reason why you'd want to use both? I am not sure how we can do that in store though.